cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -33,6 +33,7 @@ interface Props {
     selectedStatesID: number[];
     annotations: any[];
     frameData: any;
+    frameAngles: number[];
```

Comment:
I suppose canvas wrapper doesn't need angles for all frames
Only for the current one